README.md

```diff
@@ -61,6 +61,54 @@ get "/multi_auth/:provider/callback" do |env|
 end
 ```
 
+### Lucky integration example
+
+Instructions for using with [Lucky Framework](https://github.com/luckyframework/lucky).
+
+First, set up MultiAuth in a config file:
+
+```crystal
+# config/multi_auth_handler.cr
+require "multi_auth"
+
+class MultiAuthHandler
+  MultiAuth.config("facebook", "clientID", "secretKey")
+
+  getter facebook
+
+  def self.facebook
+    @@facebook ||= MultiAuth.make("facebook", "http://localhost:3000/oauth/facebook/callback")
```

Comment:
Why the domain hardcoded?

Reply:
It doesn't need to be, it could be in a config variable or something like that. I was just trying to keep it short